### DISCUSSION TITLE
TabletChecker发现到unhealth tablets之后为什么不会触发TabletScheduler？

### DISCUSSION THREAD

**Discussion by GitHubChenLin** [apache/doris#45736] [A - General / Q&A]
- 查看日志发现，TabletChecker发现了unhealth tablets后，后续为什么没有触发TabletScheduler？
![image](https://github.com/user-attachments/assets/1ee836cc-fcf4-4152-a7d8-44bde2b57e6f)
- 触发TabletScheduler的具体表现：
- 日志没有打印有关日志、使用"show proc '/cluster_health/tablet_health'" 的unhealthy tablets的数量也没有变少。
- 而重启FE之后，问题就解决了，日志中也能查到关键字眼、tablet也全都正常了：
![image](https://github.com/user-attachments/assets/7a8b1265-fcde-4278-9cbb-8054793a7c7f)
想要知道其中发生了什么

**Comment by ixzc** [comment]
maybe the show result is delayed
